types/webpack-bundle-analyzer/index.d.ts

```diff
@@ -216,7 +143,16 @@ export namespace BundleAnalyzerPlugin {
 }
 
 export class BundleAnalyzerPlugin implements WebpackPluginInstance {
+    opts: BundleAnalyzerPlugin.Options;
+    compiler?: Compiler;
+    server: null | Server;
+
     constructor(options?: BundleAnalyzerPlugin.Options);
 
     apply(compiler: Compiler): void;
+    startAnalyzerServer: (stats: WebpackStats) => Promise<void>;
+    generateJSONReport: (stats: WebpackStats) => Promise<void>;
```

Comment:
```suggestion
    /** @async */
    generateJSONReport: (stats: WebpackStats) => Promise<void>;
```
https://jsdoc.app/tags-async.html
TS does not support notion of 'async' in declaration files